test/kennel/models/slo_test.rb

```diff
@@ -11,12 +11,17 @@ class TestSlo < Kennel::Models::Slo
 
   def slo(options = {})
     Kennel::Models::Slo.new(
+      # Determine the type from options or default to "metric"
+      type = options[:type] || "metric"
       options.delete(:project) || project,
-      {
-        type: -> { "metric" },
+      slo_options ={
+        type: -> { type },
         name: -> { "Foo" },
         kennel_id: -> { "m1" }
-      }.merge(options)
+      }
+      # Conditionally include sliSpecification if type is "time_slice"
+      slo_options[:sliSpecification] = -> { options[:sliSpecification] } if type == "time_slice"
```

Comment:
idk why this is needed, we can always set it 🤷

Reply:
Without this statement I can not generate SLO for time series, I return me error that: 

```
rake aborted!
ArgumentError: 'sliSpecification' on Kennel::Models::Slo was not set or passed as option for project knowledge-api on .../knowledge_api/knowledge_api.rb:146:in `new'
```

Reply:
then the call in 88 needs to be `sliSpecification: -> { sli_spec }`